docs/proposals/work-pause/README.md

```diff
@@ -0,0 +1,500 @@
+
+## Design Details
+
+<!--
+当前，我提出了两种方案来实现当前特性，请大家给出一些具体的意见，也欢迎大家提出新的方案。让我们一同来选择一种方案，推进该特性向前发展。
+-->
+
+Currently, I've proposed two approaches to implement the current feature, and I'd appreciate concrete feedback from all of you. I also welcome suggestions for alternative solutions. Let's collaborate to select an approach and move forward with developing this feature.
+
+### Solution one
+
+<!--
+在 PP 上增加字段来描述各种暂停需求，包括：PP功能暂停、分发功能暂停、调度功能暂停、TODO：指定集群暂停。
+
+对于 Binding 来说，暂停包含：分发功能暂停、调度功能暂停、TODO：指定集群暂停；
+对于 Work 来说，暂停包含：分发功能暂停。
+
+通过扩展 PropagationPolicy/ClusterPropagationPolicy API，新增相关的暂停结构体，并将关键的暂停信息透传到 ResourceBinding/ClusterResourceBinding 资源，进而透传至 Work 资源上，Karmada-controller-manager 组件中的控制器会根据 Work 资源中的暂停信息来决定是否将资源同步至目标成员集群中去。
+-->
+
+By extending the `PropagationPolicy/ClusterPropagationPolicy` API, we introduce new suspension structures and pass critical suspension information through to the `ResourceBinding/ClusterResourceBinding` resources, which in turn are propagated to the `Work` resources. The controller within the `Karmada-controller-manager` component will then determine whether to synchronize resources to target member clusters based on the suspension information found in the `Work` resources.
+
+#### API Change
+
+```go
+// PropagationSpec represents the desired behavior of PropagationPolicy.
+type PropagationSpec struct {
+	...
+
+	// SuspendStrategy is used to specify the suspension strategy.
+	SuspendStrategy *SuspendStrategy `json:"suspendStrategy,omitempty"`
+}
+
+type SuspendStrategy struct {
+	// ResourceMatchSuspension is used to specify the suspension condition for resource
+	// matching under the current Policy, set to false indicates the suspension is canceled.
+	// Default value is false.
+	// Note: The extension design is not a focal point of the current proposal.
+	ResourceMatchSuspension *bool `json:"ResourceMatchSuspension,omitempty"`
+
+	// ScheduleSuspension is used to specify the suspension scheduling of resources
+	// matched by the current policy, set to false indicates the suspension is canceled.
+	// Default value is false.
+	// Note: The extension design is not a focal point of the current proposal.
+	ScheduleSuspension *bool `json:"scheduleSuspension,omitempty"`
+
+	// DistributionSuspendClusters is used to specify a list of clusters where resource
+	// synchronization needs to be paused. Clusters in this list indicate that resource
+	// synchronization for targeted resources will be suspended to these clusters.
+	// Clusters removed from this list indicate that resource synchronization should be resumed.
+	// If there is a cluster element with an asterisk "*" in the list, it indicates that
+	// the synchronization of resources to all clusters is paused for the matched resources.
+	DistributionSuspendClusters []string `json:"distributionSuspendClusters,omitempty"`
```

Comment:
Just one question: Why doesn't it support labelSelector like ClusterAffinity?

Reply:
Update it with a new structure. Please take a look again.